loom/graphics/gfxGraphics.h

```diff
@@ -321,14 +321,14 @@ do { \
 			const char* errorName; \
 			switch (status) { \
 				case GL_INVALID_ENUM: errorName = "GL_INVALID_ENUM"; break; \
-				case GL_FRAMEBUFFER_UNDEFINED: errorName = "GL_FRAMEBUFFER_UNDEFINED"; break; \
+				case 0x8219 /* GL_FRAMEBUFFER_UNDEFINED */: errorName = "GL_FRAMEBUFFER_UNDEFINED"; break; \

```

Comment:
THANKS OPENGL

Luckily the constant values are a part of the spec so this is OK. might be good to note that we do this and why, and why it's OK.